src/main/reactjs/src/redux/selectors/clerkTranslator.ts

```diff
@@ -64,13 +89,23 @@ export const selectFilteredSelectedTranslators = createSelector(
 
 // Helpers
 
-const isAuthorisationValid = (authorisation: Authorisation, now: Date) => {
+const expiringSoonTreshold = (currentDate: Date) => {
+  const expiringSoonThreshold = DateUtils.dateAtStartOfDay(currentDate);
+  expiringSoonThreshold.setMonth(expiringSoonThreshold.getMonth() + 3);
+
+  return expiringSoonThreshold;
+};
+
+const isAuthorisationValid = (
+  authorisation: Authorisation,
+  currentDate: Date
+) => {
   const term = authorisation.effectiveTerm;
   if (!term || !term.end) {
     return true;
   }
 
-  return now < term.end;
+  return currentDate <= term.end;
```

Comment:
This should possibly be `return currentDate <= DateUtils.dateAtStartOfDay(term.end);`

Reply:
Perhaps. The current code does rely on the fact that the date strings returned from the backend do not have a time part and thus this should produce identical results to your suggestion.